jest/jest.config.js

```diff
@@ -2,7 +2,7 @@
 // https://jestjs.io/docs/en/configuration.html
 
 // Packages from `node_modules` that explicitly need to be transpiled
-const esModules = ['common-tags'];
+// const esModules = [];
```

Comment:
If, in the future, you use npm packages that can't be read by Jest without Babel translation, uncomment and add them here.

Reply:
Thanks, I extended the help text a bit in https://github.com/FixMyBerlin/fixmy.frontend/pull/912/commits/3d0c05fca7cf2ef9f21e84bd120327bae49f9b53.